tutorials/multi/2_node_classification.py

```diff
@@ -0,0 +1,257 @@
+"""
+Single Machine Multi-GPU Minibatch Node Classification
+======================================================
+
+In this tutorial, you will learn how to use multiple GPUs in training a
+graph neural network (GNN) for graph classification.
+
+(Time estimate: 8 minutes)
+
+This tutorial assumes that you have read the :doc:`Training GNN with Neighbor
+Sampling for Node Classification <../large/L1_large_node_classification>`
+tutorial. It also assumes that you know the basics of training general
+models with multi-GPU with ``DistributedDataParallel``.
+
+.. note::
+
+   See :doc:`the multi-GPU graph classification
+   tutorial <1_graph_classification>`
+   for an overview of using ``DistributedDataParallel`` with DGL.
+
+"""
+
+
+######################################################################
+# Loading Dataset
+# ---------------
+# 
+# OGB already prepared the data as DGL graph. The following code is
+# copy-pasted from the :doc:`Training GNN with Neighbor Sampling for Node
+# Classification <../large/L1_large_node_classification>`
+# tutorial.
+# 
+
+import dgl
+import torch
+import numpy as np
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.nn import SAGEConv
+from ogb.nodeproppred import DglNodePropPredDataset
+import tqdm
+import sklearn.metrics
+
+dataset = DglNodePropPredDataset('ogbn-arxiv')
+
+graph, node_labels = dataset[0]
+# Add reverse edges since ogbn-arxiv is unidirectional.
+graph = dgl.add_reverse_edges(graph)
+graph.ndata['label'] = node_labels[:, 0]
+
+node_features = graph.ndata['feat']
+num_features = node_features.shape[1]
+num_classes = (node_labels.max() + 1).item()
+
+idx_split = dataset.get_idx_split()
+train_nids = idx_split['train']
+valid_nids = idx_split['valid']
+test_nids = idx_split['test']
```

Comment:
Not used

Reply:
I kept it regardless but added a comment saying that it is not used.